pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -463,11 +463,24 @@ func (vc *VirtualCenter) getDatacenters(ctx context.Context, dcPaths []string) (
 	finder := find.NewFinder(vc.Client.Client, false)
 	var dcs []*Datacenter
 	for _, dcPath := range dcPaths {
-		dcObj, err := finder.Datacenter(ctx, dcPath)
-		if err != nil {
-			log.Errorf("failed to fetch datacenter given dcPath %s with err: %v", dcPath, err)
-			return nil, err
+		var dcObj *object.Datacenter
+		var err error
+		dcRef := object.ReferenceFromString(dcPath)
```

Comment:
dcRef will not be nil if dcPath has ":"

```
	s := strings.SplitN(o, ":", 2)

	if len(s) != 2 {
		return false
	}
``` 

what is the expected value we need to set in the config secret to fetch datacenter using moid.

Can you share example?